model/openapi.yml

```diff
@@ -522,6 +522,11 @@ components:
         id:
           description: The policy's ID.
           type: string
+        secret_paths:
```

Comment:
Since this is outside of the signed section of the policy, in theory it could be modified to cause agent to not redact secrets locally couldn't it?

To put it in the signed section of the policy, we'd have to determine which fields contain secrets in Kibana instead, because that is where the signing key resides.

Is there a reason why we can't put these in Kibana?

Reply:
OK, thinking more deeply, I was thinking through a scenario where someone had intercepted the checkin response, and wanted to to force the agent to disclose secrets by removing the redaction hints.

However, if they have the checkin response after Fleet Server has touched it and added these keys, they already have the replaced secret values that Fleet Server inserted.

So I don't think there is a point in doing this because it doesn't protect us from anything.